src/components/register/RegisterMethods.vue

```diff
@@ -6,15 +6,26 @@ import NearIcon from '@/components/icons/brands/Near.vue';
 import InfoIcon from '@/components/icons/Info.vue';
 import CloseIcon from '@/components/icons/CloseIcon.vue';
 
+import '@near-wallet-selector/modal-ui-js/styles.css';
+import { loginWithTorus, modal } from '@/plugins/loginMethods';
```

Comment:
export 'loginWithTorus' (imported as 'loginWithTorus') was not found in '@/plugins/loginMethods' (possible exports: default)

I think you have to export it or call these methods from the `useLogin`